mozilla/bedrock
# [bug 1422899] Remove old firefox/new template

## Description
- Removes old `/firefox/new/` template.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1422899

## Testing
- Probably a good idea to run `gulp static:clean` prior to testing this PR.
- Check I didn't remove any other assets that may still be needed?
- Check translations still appear as expected.
